doc/source/getting_started/index.rst

```diff
@@ -35,18 +35,26 @@ Users can also install Covalent as a package in a Conda environment:
 Docker Install
 --------------
 
-Covalent is also provided as a Docker image. This image can be used as a developer environment or simply to run the Covalent dispatch server in a container:
+Covalent is also provided as a Docker image. You can pull the covalent server image with the following command: 
 
 .. code:: bash
 
    docker pull public.ecr.aws/i9y7r1d8/covalent
```

Comment:
```suggestion
   docker pull public.ecr.aws/covalent/covalent
```

Our custom ECR public alias was approved so we should stop using this programmatic default

Reply:
Gotcha 👍